publishers/communities/sig-cli/render-helm-chart/krm-function-metadata.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: Add CI to validate the metadata here.
+
+apiVersion: config.kubernetes.io/v1alpha1
+kind: KRMFunctionDefinition
+spec:
+  group: config.kubernetes.io
+  description: "The `render-helm-chart` function renders a local or remote Helm chart."
+  publisher: https://github.com/kubernetes/community/tree/master/sig-cli
+  names:
+    kind: RenderHelmChart
+  versions:
+    name: v1
+    idempotent: true
+    license: Apache 2.0
+    maintainers:
+      - natashasarkar@google.com
+      - mengqiy@google.com
+      - kn.verey@shopify.com
+    usage: https://github.com/kubernetes-sigs/krm-functions-registry/blob/main/krm-functions/sig-cli/render-helm-chart/README.md
+    examples:
+      - https://github.com/kubernetes-sigs/krm-functions-registry/blob/main/krm-functions/sig-cli/render-helm-chart/examples/render-helm-chart-kustomize-inline-values/README.md
+      - https://github.com/kubernetes-sigs/krm-functions-registry/blob/main/krm-functions/sig-cli/render-helm-chart/examples/render-helm-chart-kustomize-values-files/README.md
+    runtime:
+      container:
+        image: us.gcr.io/k8s-artifacts-prod/krm-functions/render-helm-chart:unstable
+        requireNetwork: true
+        requireStorageMount: true
+    schema: |-
```

Comment:
In the KEP we have `openAPIV3Schema` under this like CRD does, and then of course the schema is in OpenAPI V3 format. Is there a particular reason you're using Swagger 2.0 here?

Reply:
Ah I must have misread the KEP. I have updated it accordingly.